var/spack/repos/builtin/packages/rtmpdump/package.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Rtmpdump(MakefilePackage):
+    """rtmpdump is a toolkit for RTMP streams."""
+
+    homepage = "https://rtmpdump.mplayerhq.hu/"
+    git = "https://git.ffmpeg.org/rtmpdump.git"
+
+    maintainers("tobbez")
+
+    license("GPL-2.0-or-later")
+
+    version("2021-02-19", commit="f1b83c10d8beb43fcc70a6e88cf4325499f25857")
```

Comment:
They no longer make releases? I guess it's fine to use the last commit.

Reply:
Yes. Last release was in 2011.

At this point, others packaging rtmpdump seem to be using git snapshots.